packages/neon_framework/lib/src/widgets/unified_search_results.dart

```diff
@@ -61,53 +50,62 @@ class NeonUnifiedSearchResults extends StatelessWidget {
     AccountsBloc accountsBloc,
     UnifiedSearchBloc bloc,
     core.UnifiedSearchProvider provider,
-    Result<core.UnifiedSearchResult> result,
-  ) {
-    final entries = result.data?.entries ?? <core.UnifiedSearchResultEntry>[];
-    return Card(
-      child: Container(
-        padding: const EdgeInsets.all(10),
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            Text(
-              provider.name,
-              style: Theme.of(context).textTheme.headlineSmall,
-            ),
-            NeonError(
-              result.error,
-              onRetry: bloc.refresh,
-            ),
-            NeonLinearProgressIndicator(
-              visible: result.isLoading,
-            ),
-            if (entries.isEmpty) ...[
-              AdaptiveListTile(
-                leading: const Icon(
-                  Icons.close,
-                  size: largeIconSize,
-                ),
-                title: Text(NeonLocalizations.of(context).searchNoResults),
-              ),
-            ],
-            for (final entry in entries) ...[
-              AdaptiveListTile(
-                leading: NeonImageWrapper(
-                  size: const Size.square(largeIconSize),
-                  child: _buildThumbnail(context, accountsBloc.activeAccount.value!, entry),
-                ),
-                title: Text(entry.title),
-                subtitle: Text(entry.subline),
-                onTap: () async {
-                  context.go(entry.resourceUrl);
-                },
+  ) =>
+      StreamBuilder(
+        stream: bloc.results.map((results) => results[provider.id]),
+        builder: (context, snapshot) {
```

Comment:
Please make this a ResultBuilder
```suggestion
ResultBuilder<core.UnifiedSearchResult>(
      stream: bloc.results.map((results) => results[provider.id]!),
      builder: (context, result) {
```

Reply:
This is not going to work. Sorry